### PR TITLE
Restructure the namespace

### PR DESCRIPTION
**Pull request by jutaro**
Restructure the namespace of trace messages to get some logic into it.

### CODE REVIEW

File: trace-resources/src/Cardano/Logging/Resources/Types.hs [hunk 1]
```diff
@@ -118,12 +118,12 @@ instance LogFormatting ResourceStats where
...
-      [ IntM "stat.cputicks" (fromIntegral $ rCentiCpu rs)
-      , IntM "mem.resident" (fromIntegral $ rRSS rs)
-      , IntM "rts.gcLiveBytes" (fromIntegral $ rLive rs)
-      , IntM "rts.gcMajorNum" (fromIntegral $ rGcsMajor rs)
-      , IntM "rts.gcMinorNum" (fromIntegral $ rGcsMinor rs)
-      , IntM "rts.gcticks" (fromIntegral $ rCentiGC rs)
-      , IntM "rts.mutticks" (fromIntegral $ rCentiMut rs)
-      , IntM "rts.stat.threads" (fromIntegral $ rThreads rs)
+      [ IntM "Resources.Stat.Cputicks" (fromIntegral $ rCentiCpu rs)
+      , IntM "Resources.mem.Resident" (fromIntegral $ rRSS rs)
```
Comment (denisshevchenko) [inline trace-resources/src/Cardano/Logging/Resources/Types.hs:122]: @jutaro Is it `"Resources.mem.Resident"` and not `"Resources.Mem.Resident"`?

Comment (jutaro) [reply trace-resources/src/Cardano/Logging/Resources/Types.hs]: Fixed it

Comment (jutaro) [reply trace-resources/src/Cardano/Logging/Resources/Types.hs]: Should be: Resources.Mem.Resident

Comment (jutaro) [reply trace-resources/src/Cardano/Logging/Resources/Types.hs]: Thanks

**Review by Jimbo4350 — APPROVED** [review]

**Comment by jutaro** [comment]
bors merge

File: cardano-node/src/Cardano/Node/Tracing/Tracers/NodeToClient.hs [hunk 2]
```diff
@@ -171,54 +171,54 @@ docTChainSync = Documented [
...
+        "Acknowledge the request but require the consumer to wait for the next \
+        \update. This means that the consumer is synced with the producer, and \
         \the producer is waiting for its own chain state to change."
     , DocMsg
         ["RollForward"]
         []
-        "Tell the consumer to extend their chain with the given header.\
-        \\n\
+        "Tell the consumer to extend their chain with the given header. \
+        \\n \
```
Comment (newhoggy) [inline cardano-node/src/Cardano/Node/Tracing/Tracers/NodeToClient.hs:186]: Is the space after the newline intentional?

File: cardano-node/src/Cardano/Node/Tracing/Tracers/NodeToNode.hs [hunk 3]
```diff
@@ -472,66 +469,66 @@ docTTxSubmissionNode' = Documented [
...
+        \The order in which these transaction identifiers are returned must be \
+        \the order in which they are submitted to the mempool, to preserve \
         \dependent transactions."
     , DocMsg
         ["RequestTxs"]
         []
-        "Request one or more transactions corresponding to the given \
-        \transaction identifiers. \
-        \\n\
+        "Request one or more transactions corresponding to the given  \
```
Comment (newhoggy) [inline cardano-node/src/Cardano/Node/Tracing/Tracers/NodeToNode.hs:529]: Is the double space intentional?

File: doc/new-tracing/tracers_doc_generated.md [hunk 4]
```diff
@@ -3817,14 +3393,17 @@ Backends:
...
-### ConnectionManager.Connect
+Filtered  by config value: `Info`
 
+### Forge.Loop.ForgedBlock
 
-***
 
-***
+> We forged a block.
+>   We record the current slot number, the point of the predecessor, the block  itself, and the total size of the mempool snapshot at the time we produced  the block (which may be significantly larger than the block, due to  maximum block size)
```
Comment (newhoggy) [inline doc/new-tracing/tracers_doc_generated.md:3402]: Can see the double space manifest in this text.  Is this intentional?

**Pull request merged**